merlin/systems/dag/ops/faiss.py

```diff
@@ -95,14 +203,44 @@ def compute_input_schema(
     def compute_output_schema(
         self, input_schema: Schema, col_selector: ColumnSelector, prev_output_schema: Schema = None
     ) -> Schema:
+        """
+        Compute the input schema of this node given the root, parents and dependencies schemas of
+        all ancestor nodes.
+
+        Parameters
+        ----------
+        input_schema : Schema
+            The schema representing the input columns to the graph
+        col_selector : ColumnSelector
+            A column selector representing a target subset of columns necessary for this node's
+            operator
+        prev_output_schema : Schema
+            A schema representing the output of the previous node.
+
+        Returns
+        -------
+        Schema
+            A schema object representing all outputs of this node.
+        """
         return Schema(
             [
                 ColumnSchema("candidate_ids", dtype=np.int32),
             ]
         )
 
 
-def setup_faiss(item_vector, output_path):
+def setup_faiss(item_vector, output_path: str):
+    """
+    Utiltiy function that will create a Faiss index from an embedding vector. Currently only
```

Comment:
Utility (typo) 

Maybe "Utility function that creates a Faiss index..." (Prefer present tense.)

Up to you whether to remove "Currently."  My sugg is to keep the crystal balls far away from the software.